docs/resources/denylist.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: "wallarm"
+page_title: "Wallarm: wallarm_denylist"
+subcategory: "Common"
+description: |-
+  Provides the resource to manage denylist in the account.
+---
+
+# wallarm_denylist
+
+Provides the resource to manage [denylist][1] in the account providing functionality to block all requests originated from denylisted IP addresses for a desired time either via `Date` or `Minutes` formats.
```

Comment:
the examples below does not mention "Date" as a value of time_format, it mentions RFC3339, so it is better to either delete code formatting from "Date" or mention the possible values of time_format as is

Reply:
Nice catch. It should be `RFC3339`, not `Date`.